galoy.yaml

```diff
@@ -129,6 +129,7 @@ accountLimits:
       "2": 20000000
 spamLimits:
   memoSharingSatsThreshold: 1000
+  memoSharingCentsThreshold: 50
```

Comment:
This seems high, any particular reason for this?

Reply:
was the value in the previous PR.. nothing in particular.. what value do you suggest?